draft-barnes-acme.md

```diff
@@ -1649,9 +1649,8 @@ The client serializes the validation object to UTF-8, then uses its account
 private key to sign a JWS with the serialized JSON object as its payload.  This
 JWS is NOT REQUIRED to have the "nonce" header parameter.
 
-The client will compute Z, the SHA-256 of the "signature" value from the JWS.
-The hash is calculated over the base64-encoded signature string.  Z is encoded
-in hexadecimal form.
+The client will compute Z, the SHA-256 of the JWS encoded in UTF-8.
```

Comment:
Same comment/question as for DNS: why not use simple serialization instead of JWS